sos/policies/package_managers/flatpak.py

```diff
@@ -0,0 +1,34 @@
+# Copyright 2023 Red Hat, Inc. Jose Castillo <jcastillo@redhat.com>
+
+# This file is part of the sos project: https://github.com/sosreport/sos
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# version 2 of the GNU General Public License.
+#
+# See the LICENSE file in the source distribution for further information.
+
+from sos.policies.package_managers import PackageManager
+
+
+class FlatpakPackageManager(PackageManager):
+    """Package Manager for Flatpak distributions
+    """
+
+    query_command = 'flatpak list --columns=name,version,branch'
+    query_path_command = ''
+    files_command = ''
+    verify_command = ''
+    verify_filter = ''
+
+    def _parse_pkg_list(self, pkg_list):
+        for line in pkg_list.splitlines()[1:]:
+            pkg = line.split("\t")
+            name, branch = pkg[0], pkg[2]
+            if not pkg[1]:
+                version = ''
```

Comment:
From what I've found in my tests, the column for 'version' can be empty. I haven't found if other values can be as well, but installing multiple packages have shown that Name and Branch are never empty.

`$ flatpak list --columns=name,version,brancholicy 
Name                        Version         Branch
Podman Desktop              1.2.0           main
Freedesktop Platform        22.08.14        22.08
Mesa                        23.1.4          22.08
Mesa (Extra)                23.1.4          22.08-extra
Intel                                       22.08
openh264                    2.1.0           2.2.0
`

Reply:
I just saw that as per my installation of flatpak too

Reply:
Values which are filled from appdata file like Version and Description can be empty.
Other values are filled by flatpak-builder so they can't be empty, because if you don't put any value there, flat-builder doesn't build the package.